tests/TranslatableTest.php

```diff
@@ -608,4 +620,39 @@ public function it_can_replace_translations()
 
         $this->assertEquals($newTranslations, $this->testModel->getTranslations('name'));
     }
+
+    /** @test */
+    public function it_can_translate_json_fields()
+    {
+        $en = [ 'text' => 'hello' ];
```

Comment:
Let's use full variable names here. Rename `$en` to `$englishTranslation`.

Reply:
Done